src/preferences/beatdetectionsettings.h

```diff
@@ -64,14 +64,14 @@ class BeatDetectionSettings {
             plugin_id);
     }
     QString getBeatPluginIdDefault() const {
-        return "qm-tempotracker";
+        return "qm-tempotracker:0";
```

Comment:
The default is actually `AnalyzerBeats::availablePlugins().front().id` to avoid redundant default definitions